### DISCUSSION TITLE
Only 15GB disk after installation from Anaconda via kickstart

### DISCUSSION THREAD

**Discussion by pbrezina** [bootc-dev/bootc#967] [Q&A] [answered]
I used the following kickstart to trigger a bootc installation via Anaconda
```
ostreecontainer --url quay.io/pbrezina/fedora-bootc-workstation:latest
```

The image is based on fedora-bootc:latest with workstation environment installed in it. During installation, I chose automatic partitioning on my 1TB drive. It seemed to work fine but then I tried to `bootc upgrade` which failed due to lack of space on the disk and when I checked, there is only `/dev/mapper/fedora-root` (besides all the tmpfs stuff) which has only 15GB.

Is this expected? Should I just use manual partitioning and if yes what partitions should I create?

**Comment by cgwalters** [comment]
The Anaconda partitioning defaults are just plain broken. See e.g. https://issues.redhat.com/browse/RHEL-56630

But sorry it's nothing related to bootc, sorry. Chiming in on that issue is probably a good place. Or maybe fedora-devel@?

**Comment by pbrezina** [comment, marked as answer]
Thank you. The answer is to ignore the auto partitioning for now and just use manual and create on big root (I did not try creating more partitions, maybe that would work too).
